README.md

```diff
@@ -4,3 +4,43 @@
+- **Generate_Bids**: (optional) Set as a configuration setting.  If true, this gear will also provide BIDS compliant .tsv.gz files
+
+## Function:
+
+This gear uses the [CMRRExtractPhysio](https://github.com/CMRR-C2P/MB/) program to exctract physiological data from the dicom to separate .log files.  These log files are then used to generate visual plots of the physiological recording, with indicator lines marking the beginning and end of the scan, for validation purposes.
+
+If **Generate_Bids** is set to True, then two additional files are created for each physio .log file generated from CMRRExtractPhysio, following the [BIDS naming conventions](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/06-physiological-and-other-continous-recordings.html).
+
+While BIDS allows multiple physiological recordings to be placed in the same .tsv.gz file, this can only be done if they have the same sampling rate.  Typically, this is not the case.  Because of this, this gear always creates individual .tsv.gz files for each physiological recording.  
+
+BIDS also allows for a "scanner trigger" column in each recording's .tsv.gz file.  Though not explicitly stated, this logically is referring to the trigger that is sent by the scanner when a new volume acquisition begins.  This column is included with each physiological recording, synced to each individual sampling rate and time.
+
+This gear looks for metadata info on the acquisition name for the BIDS naming convention.  If this isn't availible, it will try to pull the "SeriesDescription" tag directly from the header.  Missing metadata in both of these locations will result in an output file named "UnknownAcquisition_<Measurement>.tsv.gz", and the user will need to manually set these file names.
+  
+
+## Outputs:
+
+Every physiological dicom generates one "info.log" file, which has information about the acquisition time of the volumes in the scan.  This contains no physiological information, but is necessary to synchronize the physiological recordings with the scan.
+
+An additional "<Measure>.log" file is created for each physiological measurment stored in the dicom.
+A validation .png image is generated for each "<Measure>.log" file.
+  
+If BIDS generation is selected, an additional ".tzv.gz" and ".json" file are created for each "<Measure>.log" file.
+The following directory structure represents output for a dicom with a single physiological measurement (respiration: "RESP").  Files surrounded by "[]" indicate files that are only generated for BIDS.  Files surrounded by "{}" indicate files that will be generated for each individual physiological measurement, if present:
+
+```
+Output_Directory
+|
+|--->     Physio_..._info.log
+|--->   { Physio_..._RESP.log         }
+|--->   { RESP.png                    }
+|---> [ { BIDS_name_for_RESP.tsv.gz   } ]
+|---> [ { BIDS_name_for_RESP.json     } ]
+
+
```

Comment:
Perhaps include a description (best practice) of how to validate the measurements like you did when you showed one to us.  What are you looking for?  What makes it valid and what can go wrong?